pkg/services/rendering/interface.go

```diff
@@ -11,7 +11,15 @@ import (
 var ErrTimeout = errors.New("timeout error - you can set timeout in seconds with &timeout url parameter")
 var ErrPhantomJSNotInstalled = errors.New("PhantomJS executable not found")
 
+type RenderType string
```

Comment:
AFAICT this doesn't have to be exported:
```suggestion
type renderType string
```

Reply:
It's needed in the Enterprise repo.